ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go

```diff
@@ -46,7 +46,7 @@ func ResourceIbmOnboardingIamRegistration() *schema.Resource {
 			},
 			"name": &schema.Schema{
 				Type:         schema.TypeString,
-				Optional:     true,
+				Required:     true,
```

Comment:
Changing from Optional to Required are kind of breaking changes for user's . IF its required share me notes section which I can included in release notes

Reply:
All of our documents contains the following notice:
**Note - Intended for internal use only. This resource is strictly experimental and subject to change without notice.**